### PR TITLE
Maintenance: Switch from scheduler.rb to background-services.rb

### PR DESCRIPTION
**Pull request by MrGeneration**

### CODE REVIEW

File: containers/zammad/setup.sh
Comment (mgruner) [inline containers/zammad/setup.sh:47]: I believe this line can be dropped, as the new background-worker.rb does not write own logs any more.

**Review by mgruner — APPROVED** [review]
Please see one small suggestion, clean-up possible. 

There is still the usage of `zammad-scheduler` as a container name and entry point argument. I assume you want to keep that to avoid breakage, right?

**Review by monotek — COMMENTED** [review]
Is this needed for 5.2 or do both implementations work for now?

If its needed it should be merged and we would have to wait for next docker build before we merge: https://github.com/zammad/zammad-docker-compose/pull/283

**Comment by MrGeneration** [comment]
This is needed for Zammad 5.2 and does not work with 5.1.

I'm a little confused on how the built process works for compose and what exactly triggers it.
Tag wise we are already on 5.2.x (if that's relevant to your above PR note)

image

**Comment by mgruner** [comment]
> Is this needed for 5.2 or do both implementations work for now?
> 
> If its needed it should be merged and we would have to wait for next docker build before we merge: #283

@monotek Zammad 5.2 still works with the (deprecated) Scheduler, but that will be removed with Zammad 6.0. So right now, both methods work.

**Comment by monotek** [comment]
@MrGeneration 
The build is triggered if there is a new commit in the stable branch of zammad/zammad.

**Review by monotek — APPROVED** [review]

**Comment by monotek** [comment]
Can we merge this one?

**Comment by monotek** [comment]
In the end it's not that important but I would merge this one first.

**Pull request merged**
